nncf/openvino/rt_info.py

```diff
@@ -17,6 +17,23 @@
 from nncf.scopes import IgnoredScope
 
 
+def handle_ignored_scope(value: IgnoredScope) -> IgnoredScope:
```

Comment:
It would be great to have here less general name. For example, the `exclude_empty_fields()` name gives us some information about what this function will do.